pkg/ddl/index.go

```diff
@@ -2563,3 +2565,11 @@ func renameIndexes(tblInfo *model.TableInfo, from, to model.CIStr) {
 		}
 	}
 }
+
+func renameHiddenColumns(tblInfo *model.TableInfo, from, to model.CIStr) {
+	for _, col := range tblInfo.Columns {
+		if strings.Contains(col.Name.O, fmt.Sprintf("%s_%s", expressionIndexPrefix, from)) {
```

Comment:
How about `abc_V$_idx1_0`? Do we need to update this check like `getChangingIndexOriginName`?